### PR TITLE
Added an exclamation mark for enthusiasm

### PR DESCRIPTION
**Pull request by DarinVicaldo**

### CODE REVIEW

**Review by growlf — COMMENTED** [review]
Please always supply a Description for your PRs. Repository maintainers deal with lots of code on a constant basis, descriptions help us understand the purpose of the PR before we have to dig into it deeper - this allows the maintainer to more easily prioritize the PRs at a glance.

File: README.md
Comment (growlf) [inline README.md:1]: A good addition of documentation for something the developer added but did not document - a common issue in opensource projects.

**Comment by abarcomb-work** [comment]
The PR should have been made to the ensf400-ws24 branch, not to main.

**Comment by growlf** [comment]
This PR seems to be comprised of code and changes that already exist in other branches, and so does not contribute anything new.

**Pull request closed without merging**